event/management/commands/instagram_backfill.py

```diff
@@ -0,0 +1,86 @@
+import requests
+import time
+import environ
+import random
+import re
+from scrapy.selector import Selector
+
+from django.core.management.base import BaseCommand
+
+from event.models import Artist, Instagram
+
+env = environ.Env()
+
+def parse_stat(formatted_stat):
+    if (not len(formatted_stat)):
+        return None
+
+    if (formatted_stat[-1] == "M"):
+        return int(formatted_stat[:-1].replace(',','')) * 1_000_000
```

Comment:
optional: instead of `replace + int`, you can use

https://stackoverflow.com/questions/1779288/how-to-convert-a-string-to-a-number-if-it-has-commas-in-it-as-thousands-separato